globus_automate_client/cli/flows.py

```diff
@@ -327,9 +432,9 @@ def flow_update(
             subtitle,
             description,
             keywords,
-            visible_to,
-            runnable_by,
-            administered_by,
+            flow_viewer,
```

Comment:
It looks like we end up not using any of the aliases in the actual SDK call. Should we incorporate the aliases somehow? I'm ok if we decide to drop the aliases in this release.